zephyr/CMakeLists.txt

```diff
@@ -263,6 +263,13 @@ if (CONFIG_SOC_SERIES_INTEL_ADSP_ACE)
 		)
 	endif()
 
+	if (CONFIG_SOC_INTEL_ACE30_PTL)
+		zephyr_library_sources(
```

Comment:
could use `zephyr_library_sources_ifdef(CONFIG_SOC_INTEL_ACE30_PTL)`, similar in MTPM and LNL cases above too